scripts/train_bertmesh_script.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
```

Comment:
we do not want another script, we want dvc to run the train file, this is redundant. remove.

Reply:
Removed